test/libsolidity/semanticTests/array/copying/function_type_array_to_storage.sol

```diff
@@ -46,7 +46,7 @@ contract C {
 }
 // ----
 // test() -> 0x20, 0x14, "[a called][b called]"
-// gas irOptimized: 116673
+// gas irOptimized: 116660
```

Comment:
Ok, this is weird. The gas savings come from the fact that the optimizer removes an `mstore(0, 1)`. It is correct to remove it but I have absolutely no idea why the old version was not able to do that. The mstore (is / in the end ends up) in the root ast node (i.e. is inlined into the external dispatch), so this might very well be fixed by https://github.com/ethereum/solidity/pull/12769 as well.